src/renderer/assets/styles/publication.css

```diff
@@ -164,3 +164,39 @@
         }
     }
 }
+
+.cover {
+    height: 100%;
+    width: 100%;
+    text-align: center;
+    box-sizing: border-box;
+    padding: 5%;
+    background-image: linear-gradient(#d18e4b, #7c4c1c);
```

Comment:
Suggestion: use CSS variables in `background-image: linear-gradient(#d18e4b, #7c4c1c);`, so that randomly-generated colour schemes can be used (see my comment about `publication.customCover` deprecation).